### PR TITLE
ASC-398 add maketasks for release

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,3 +96,30 @@ develop: clean ## install necessary packages to setup a dev environment
...
+	echo 'Successfully released!'
+	echo 'Please push the newly created tag and commit to GitHub.'
+
+release-minor: clean-build build develop bump-minor lint test publish ## package and upload a minor release
+	echo 'Successfully released!'
+	echo 'Please push the newly created tag and commit to GitHub.'
+
+release-patch: clean-build build develop bump-patch lint test publish ## package and upload a patch release
+	echo 'Successfully released!'
+	echo 'Please push the newly created tag and commit to GitHub.'
```
Comment (ryan-rs) [inline Makefile]: Missing newline at EOF.

**Comment by zreichert** [comment]
@ryan-rs I'll add a link in this PR and PR a link into zigzag as well.

File: README.rst [hunk 2]
```diff
@@ -40,12 +40,18 @@ Contributing
 
 See `CONTRIBUTING.rst`_ for more details on developing for the "pytest-rpc" project.
 
+Release Process
+---------------
+
+See `docs/release_process.rst`_ for information on the release process for 'pytest-rpc'
```
Comment (ryan-rs) [inline README.rst]: Not rendering correctly because you've got `docs/release_process.rst_` instead of `release_process.rst_`.

File: README.rst [hunk 3]
```diff
@@ -40,12 +40,18 @@ Contributing
...
+
+See `release_process.rst`_ for information on the release process for 'pytest-rpc'
+
 Credits
 -------
 
 This `Pytest`_ plugin was generated with `Cookiecutter`_ along with `@hackebrot`_'s `Cookiecutter-pytest-plugin`_ template.
 
 .. _CONTRIBUTING.rst: CONTRIBUTING.rst
+.. _release_process.rst: docs/release_process.rst
```
Comment (ryan-rs) [inline README.rst]: References "docs" when the directory is actually "doc".

**Pull request merged**